react/src/pages/datasets/DatasetTable.tsx

```diff
@@ -1,9 +1,9 @@
-import React, { useState } from 'react';
-import { makeStyles, Chip, IconButton } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import { makeStyles, Chip, IconButton, TextField } from '@material-ui/core';
 import { PlayArrow, Delete, Cancel } from '@material-ui/icons';
 import MaterialTable, { MTableToolbar } from 'material-table';
-import { toKeyValue } from "../utils";
-import AnalysisRunnerDialog from './AnalysisRunnerDialog';
+import { toKeyValue, KeyValue } from "../utils";
+import AnalysisRunnerDialog, { Pipeline } from './AnalysisRunnerDialog';
 
 export interface Dataset {
     dataset_id: number;
```

Comment:
Maybe we should move the interfaces for Dataset, Participant, etc. to utils.tsx file?